### PR TITLE
add export_wb sample for 'fullpdf'

### PR DESCRIPTION
**Pull request by RussTheAerialist**
Emulating the basic functionality of fullpdf from tabcmd.

### CODE REVIEW

File: samples/export_wb.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+def cleanup(tempdir):
+    shutil.rmtree(tempdir)
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Export to PDF all of the views in a workbook')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--site', '-S', default=None)
+    parser.add_argument('-p', default=None)
```
Comment (LGraber) [inline samples/export_wb.py]: Add --password

File: samples/export_wb.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                        help='desired logging level (set to error by default)')
+    parser.add_argument('--file', '-f', help='filename to store the exported data')
+    parser.add_argument('resource_id', help='LUID for the view')
```
Comment (LGraber) [inline samples/export_wb.py]: "for the workbook"

File: samples/export_wb.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+def main():
+    parser = argparse.ArgumentParser(description='Export to PDF all of the views in a workbook')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--site', '-S', default=None)
+    parser.add_argument('-p', default=None)
+
+    parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                        help='desired logging level (set to error by default)')
+    parser.add_argument('--file', '-f', help='filename to store the exported data')
```
Comment (LGraber) [inline samples/export_wb.py]: Add to the help that this is the default or just make it explicitly the default

File: samples/export_wb.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    logging.basicConfig(level=logging_level)
+
+    tempdir = tempfile.mkdtemp('tsc')
+    logging.debug("Saving to tempdir: %s", tempdir)
+
+    tableau_auth = TSC.TableauAuth(args.username, password, args.site)
+    server = TSC.Server(args.server, use_server_version=True)
+    try:
+        with server.auth.sign_in(tableau_auth):
+            views = list(TSC.Pager(server.views))
```
Comment (LGraber) [inline samples/export_wb.py]: This seems dumb (Russell told me to put that :) )

Comment (RussTheAerialist) [reply samples/export_wb.py]: I still need to address the dumb.

**Pull request merged**